GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/SurveyDto.java

```diff
@@ -33,7 +33,7 @@
     private List<QuestionGroupDto> questionGroupList;
     private String path;
     private Long surveyGroupId = null;
-    private String pointType;
+//    private String pointType;
```

Comment:
Same here, maybe remove commented out code on this file

Reply:
Definitely